packages/react/src/components/Authenticator/shared/FormField.tsx

```diff
@@ -45,7 +45,11 @@ export function FormField({
           hasError={hasError}
           aria-describedby={ariaDescribedBy}
         />
-        <ValidationErrors errors={errors} id={errorId} />
+        <ValidationErrors
+          dataAttr="data-amplify-sign-up-errors"
```

Comment:
These `dataAttr` is specific to Authenticator, so I made this a prop.

Reply:
Isn't `ValidationErrors` only used in the Authenticator? So shouldn't it always use the `data-amplify-sign-up-errors` prop? Or are there some scenarios where you wouldn't pass this in?

Reply:
Will the Account settings be using this Validation Errors without the prop?

Reply:
They won't be for now, I don't see its styles to be too useful. Will discuss with @dbanksdesign when I make the PR (#2900) ready for review.